.gitignore

```diff
@@ -66,3 +66,6 @@ out/
 
 ### Environment Variables ###
 .env
+
+### iCal files
+.ics
```

Comment:
This is good but there are still lots of `.ics` files in the PR.
I believe it should be `*.ics`  - https://git-scm.com/docs/gitignore
and also you have to remove the already commited .ics files - https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
.ics has been updated to *.ics